src/components/virtualRepeater/virtualRepeater.scss

```diff
@@ -0,0 +1,57 @@
+$virtual-repeat-scrollbar-width: 16px;
+
+.md-virtual-repeat-container {
+  box-sizing: border-box;
+  margin: 0;
+  overflow: hidden;
+  padding: 0;
+  position: relative;
+
+  .md-virtual-repeat-scroller {
+    bottom: 0;
+    box-sizing: border-box;
+    left: 0;
+    margin: 0;
+    overflow-x: hidden;
+    overflow-y: auto;
+    padding: 0;
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
+
+  .md-virtual-repeat-sizer {
+    box-sizing: border-box;
+    height: 1px;
+    display: inline-block;
+    margin: 0;
+    padding: 0;
+    width: 1px;
+  }
+
+  .md-virtual-repeat-offsetter {
+    box-sizing: border-box;
+    left: 0;
+    margin: 0;
+    padding: 0;
+    position: absolute;
+    // Leave room for the scorll bar.
+    // TODO: Will probably need to performa measurements at runtime.
```

Comment:
I think it is. Right now the scrollbar size is assumed to be 16px, but it's not the same across browsers/OSes/etc.

(I am sure there are battle-tested ways of doing that. I think Bootstrap has a function to measure scrollbar size, which I think they have taken from somewhere else (maybe jQueryUI).)